doc tests don't pass

Looks like should be a basic scope fix. 

```
---- src/lib.rs -  (line 8) stdout ----
	error[E0425]: cannot find function `decode_block` in this scope
  --> src/lib.rs:21:13
   |
21 | let block = decode_block(b).unwrap();
   |             ^^^^^^^^^^^^ not found in this scope
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -24,7 +24,7 @@
 //!dXd/H5LMDWnonNvPCwQUHt==
 //!-----END PRIVACY-ENHANCED MESSAGE-----";
 //!
-//!let block = decode_block(b).unwrap();
+//!let block = nom_pem::decode_block(b).unwrap();
 //!
 //!assert_eq!(block.block_type, "PRIVACY-ENHANCED MESSAGE");
 //!assert_eq!(block.data.len(), 160);

